app/models/middleware_server.rb

```diff
@@ -33,7 +33,8 @@ def evaluate_alert(alert_id, event)
     s_start = event.full_data.index("id=\"") + 4
     s_end = event.full_data.index("\"", s_start + 4) - 1
     event_id = event.full_data[s_start..s_end]
-    if event_id.start_with?("MiQ-#{alert_id}") && event.middleware_server_id == id
+    if event.middleware_server_id == id && (
+      event_id.start_with?("MiQ-#{alert_id}") || event_id.start_with?(ext_management_system.miq_id_prefix))
```

Comment:
how about moving that logic to your subclass?

`ext_management_system.miq_id_prefix` is a method that is only avail in `Hawkular::MiddlewareManager`

Reply:
Yes, indeed `evaluate_alert` should be made _abstract_. All its code is specific to Hawkular. May I move the code in a follow-up PR?

BTW, the asignee didn't change.

Reply:
I'm good with a followup PR for this, I'm not a fan of checking `event_id.start_with?` maybe you can try checking the source?